fdbserver/workloads/ConsistencyCheck.actor.cpp

```diff
@@ -78,6 +78,9 @@ struct ConsistencyCheckWorkload : TestWorkload
 	// Whether to suspendConsistencyCheck
 	AsyncVar<bool> suspendConsistencyCheck;
 
+	// Storage queue size limit
+	static const int64_t kStorageQueueLimit = 10000;
```

Comment:
remove this and disable the transaction does by the backup worker (like how timekeeper is disabled)

Reply:
After talk with Alex, I realized this is not going to be a problem. In the backup simulation test, the backup is first stopped and then undergoing the consistency check. Just need to make sure the backup worker are in NOOP mode during the check.